src/common/context/forms/index.tsx

```diff
@@ -28,25 +30,32 @@ export const useFormsContext = (): FormsContext => useContext<FormsContext>(Form
 export const FormsContextProvider: FunctionComponent = ({ children }) => {
   const [activeFormIndex, setActiveFormIndex] = useState<number | undefined>(undefined);
   const [forms, setForms] = useState<FormEntry[]>([]);
+  const { config } = useConfigContext();
+
+  const currentForm = activeFormIndex === undefined ? undefined : forms[activeFormIndex];
```

Comment:
isn't `const currentForm = forms[activeFormIndex]` the same ?

Reply:
hmm apparently ts throws when there is undef in the [].

![image](https://user-images.githubusercontent.com/7406870/89155951-730bf780-d59c-11ea-850a-10e1652d0832.png)

Reply:
The alternative is then `activeFormIndex ? forms[activeFormIndex] : undefined`, but when the index = 0, then it will be undefined.